sechub-developertools/scripts/sechub-api.sh

```diff
@@ -146,6 +149,30 @@ function sechub_project_assign_user {
 }
 
 
+function generate_sechub_project_create_data {
+  local PROJECT_ID="${1,,}"  # ,, converts to lowercase
+  shift
+  local OWNER="${1,,}"
+  shift
+  local SHORT_DESCRIPTION="$*"
+  cat <<EOF
+{
+  "apiVersion":"$SECHUB_API_VERSION",
+  "name":"$PROJECT_ID",
+  "owner":"$OWNER",
+  "description":"$SHORT_DESCRIPTION"
+}
+EOF
+}
+
+function sechub_project_create {
+  generate_sechub_project_create_data $1 $2 $3  # Show what is sent
+  curl $CURL_PARAMS -i -X POST -H 'Content-Type: application/json' \
+    -d "$(generate_sechub_project_create_data $1 $2 $3)" \
```

Comment:
Instead of calling `generate_sechub_project_create_data` twice you could simply save the result in a variable and use the variable twice. In addition, a variable would make the code a bit more readable.

Reply:
Implemented.